Hotfix : reference error - CommunityView

AndresIvanMedinaH opened the pull request:
**Task’s name**
reference error - CommunityView
 **Task’s link to navigate to jira**
https://devaptivist.atlassian.net/browse/LP-179?atlOrigin=eyJpIjoiZWRkZjlmNmY0N2IyNDIyYmIyZTE2NmMwZDhiM2RmMGMiLCJwIjoiaiJ9
 **General task’s description of what was added, deleted or updated in the code.**
When trying to call the view of the Community module in the App module, that error is generated
 **Attachment file of how looks your code in the app or in the preview if it is necessary.** 
![communityUI](https://user-images.githubusercontent.com/128720218/235202904-d4f75dc7-8b1a-41e1-8c1e-0592fdb5eca4.jpg)

/******************************************/
adding description
+renamed view to CommunityView
+Update on Graddle
+Removed unused string in strings.xml
+reformat communityComposbale.kt
+removed resource to  strings file <string name="uI_title_community">Community</string>
+Used :  <string name="title_community">Community</string>

AndresIvanMedinaH committed:
LP-179-referenceError-CommunityView
# Conflicts:
Renamed reference to ID plugin com.android.library

adding description
+renamed view to CommunityView
+Update on Graddle
+Removed unused string in strings.xml
+reformat communityComposbale.kt
+removed resource to  strings file <string name="uI_title_community">Community</string>
+Used :  <string name="title_community">Community</string>

irv205 commented on feature/community/src/main/res/values/strings.xml:
```diff
@@ -5,6 +5,5 @@
     <string name="communityUi_cardButton_communityWall">Community Wall</string>
```
rename the string file, only use example:
`<string name="title">Community Wall</string>`
It is not necessary to specify which view they are from since it is a modularized app

melissaquezada94 replied:
Same for the other strings, please remove communityUI

AndresIvanMedinaH replied:
the comments are not so clear, need more information, please elaborate over the thing that needs to be updated

melissaquezada94 replied:
Please remove the communityUI from all the strings

AndresIvanMedinaH replied:
do you mean by any chance to adapt the nomenclature of each item value by removing the last part "_communityWall" from it ?  is this what you want to be done?

melissaquezada94 replied:
Just remove the communityUI from all the strings

AndresIvanMedinaH replied:
fixed  'n  pushed

irv205 replied:
remove "communityUi_", from all the strings

AndresIvanMedinaH replied:
IT was pushed yesterday at 4:00 the changes as I stated in Whatsapp coms

AndresIvanMedinaH replied:
![imagen](https://user-images.githubusercontent.com/128720218/235707213-791a22fa-884f-4e20-872c-b713b6e63ec9.png)

irv205 replied:
Your last comment was 4 days ago. 

![image](https://user-images.githubusercontent.com/42255248/235707683-6a14f367-507e-408b-958a-03a6dd66182b.png)

If the change can't be reflected in the repository, I can't verify that it actually exists.

AndresIvanMedinaH committed:
adding description
+renamed view to CommunityView
+Update on Graddle
+Removed unused string in strings.xml
+reformat communityComposbale.kt
+removed resource to  strings file <string name="uI_title_community">Community</string>
+Used :  <string name="title_community">Community</string>
Merge branch 'develop' into hotfix/reference-error-CommunityView

AndresIvanMedinaH committed:
adding description
+renamed view to CommunityView
+Update on Graddle
+Removed unused string in strings.xml
+reformat communityComposbale.kt
+removed resource to  strings file <string name="uI_title_community">Community</string>
+Used :  <string name="title_community">Community</string>
Merge branch 'develop' into hotfix/reference-error-CommunityView
